### DISCUSSION TITLE
All the edu url is redirected to github pricing.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#138348] [GitHub Education]
### Hi everyone,

I am a student at PolyU. After successfully applying for the student package last month, I suddenly lost all my rights and interests in the past few days. When I try to check, all education-related links are redirected to Github Pricing. This completely prevents me from accessing any EDU interface or viewing any information. I look forward to any help.

**Comment by davevad93** [comment]
Hi @1014982466 ,

This has also happened to other users before and they have solved it by contacting support, raise a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Reply by ghost** [reply]
> Hi @1014982466 ,
> 
> This has also happened to other users before and they have solved it by contacting support, raise a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

Thanks for your advice. But I have raised 2 tickets already. Both got "Closed" very shortly and nobody replied to me at all.

**Reply by ghost** [reply]
Why did Github just close all my Ticket without any response or troubleshooting? I can do without the EDU PACK, but can anyone at least give me a reason?
